packages/expo-dev-client-components/src/theme.ts

```diff
@@ -97,15 +97,15 @@ export const text = {
   size: {
     small: {
       fontSize: 12,
-      lineHeight: 14,
+      lineHeight: 18,
     },
     medium: {
       fontSize: 16,
-      lineHeight: 18,
+      lineHeight: 22,
     },
     large: {
       fontSize: 18,
-      lineHeight: 24,
+      lineHeight: 26,
     },
```

Comment:
It would be nice if does value came from native styleguide:
* https://github.com/expo/styleguide/blob/main/packages/styleguide-native/src/styles/typography.ts

Reply:
even better would be to import typography directly - I didn't even realize it was exported - thanks @Simek !

Reply:
Yeah, that's what I have in mind, since `@expo/styleguide-native` package is already in the dependencies. 🙂